src/main/java/wooteco/subway/admin/domain/Line.java

```diff
@@ -78,20 +94,87 @@ public void update(Line line) {
         if (line.getIntervalTime() != 0) {
             this.intervalTime = line.getIntervalTime();
         }
+        if (line.getColor() != null) {
+            this.color = line.getColor();
+        }
+        if (line.getLineStations() != null) {
+            this.lineStations = line.getLineStations();
+        }
 
         this.updatedAt = LocalDateTime.now();
     }
 
     public void addLineStation(LineStation lineStation) {
-        // TODO: 구현
+        if (lineStations.isEmpty()) {
+            validateFirstLineStation(lineStation);
+        }
+        lineStations.add(lineStation);
+
+        Long preStationId = lineStation.getPreStationId();
+        Long stationId = lineStation.getStationId();
+
+        Iterator iterator = lineStations.iterator();
+        while (iterator.hasNext()) {
+            LineStation selectedLineStation = (LineStation)iterator.next();
```

Comment:
```suggestion
        Iterator<LineStation> iterator = lineStations.iterator();
        while (iterator.hasNext()) {
            LineStation selectedLineStation = iterator.next();
```

제네릭을 이용하면 캐스팅을 제거할 수 있습니다.

Reply:
addLineStation 메서드의 로직을 바꾸고 리팩토링하였습니다! 해당 로직은 삭제하였습니다! :)